action.yml

```diff
@@ -27,9 +28,31 @@ inputs:
   pr-number:
     description: 'A pull request number, only required if triggered from a workflow_dispatch event'
     required: false
+
 runs:
-  using: 'node16'
-  main: 'dist/index.js'
+  using: 'composite'
+  steps:
+    - name: Fetch metadata
+      id: dependabot-metadata
+      uses: dependabot/fetch-metadata@v1
+    - name: Merge/approve PR
+      uses: actions/github-script@v6
+      if: ${{ github.event_name == 'pull_request' }}
```

Comment:
let's include the simple check about the dependabot user here so we avoid even executing the rest of the action if it's not dependabot. we can slightly simplify the check in the JS code then